.github/workflows/downgraded_release.yaml

```diff
@@ -33,16 +33,16 @@ jobs:
             -   run: mkdir rector-local
             -   run: composer require rector/rector --working-dir rector-local --ansi
 
-            # downgrade to PHP 7.2
-            -   run: rector-local/vendor/bin/rector process src bin vendor --config build/rector-downgrade-php-72.php --ansi
+            # downgrade to PHP 7.4
+            -   run: rector-local/vendor/bin/rector process src bin vendor --config build/rector-downgrade-php-74.php --ansi
```

Comment:
This would make this package useless for upgrade. I've got few PHP 7.2 around. Please revert this change

Reply:
lets see whether nikic/php-parser will work for 7.2 after it got downgraded